src/rewriter.ts

```diff
@@ -92,10 +92,23 @@ export abstract class Rewriter {
     this.writeRange(pos, node.getEnd());
   }
 
-  // Write a span of the input file as expressed by absolute offsets.
-  // These offsets are found in attributes like node.getFullStart() and
-  // node.getEnd().
+  private skipToOffset = 0;
+
+  /**
+   * Skip emitting any code before the given offset. Used to avoid emitting @fileoverview comments
+   * twice.
+   */
+  skipUpToOffset(offset: number) {
```

Comment:
It cannot have the same name – in JS, properties and functions are in the same namespace. I didn't want to just assign into a field across two files and a class hierarchy, but then again this is all internal to the library, and we don't really have proper separation here anyway.

So changed to a simple field.